ding/envs/env_manager/base_env_manager.py

```diff
@@ -464,6 +465,7 @@ def step(self, actions: List[tnp.ndarray]) -> List[tnp.ndarray]:
             # make the type and content of key as similar as identifier,
             # in order to call them as attribute (e.g. timestep.xxx), such as ``TimeLimit.truncated`` in cartpole info
             info = make_key_as_identifier(info)
+            info = remove_illegal_item(info)
```

Comment:
Why should we have to remove string in info?

Reply:
it(string) is not compatible with Tensor